pybossa/api/task.py

```diff
@@ -62,6 +62,10 @@ def _forbidden_attributes(self, data):
                 raise BadRequest("Reserved keys in payload")
 
     def _update_attribute(self, new, old):
+        for key, value in old.info.items():
+            if not new.info.get(key):
+                new.info[key] = value
```

Comment:
```
if not new.info.get(key):
   new.info[key] = value
```
is equivalent to ```new.info.setdefault(key, value)```. A little more concise but not required to make the changes. BTW, why do we need to copy the old key value to the new if not existed?

Reply:
copying old values when not exist makes the logic concise; client need to pass only the attribute whose value need to be modified.